codeGenerator/src/robotConfiguration/robotConfig.cs

```diff
@@ -89,26 +90,56 @@ robotVariants loadRobotConfiguration(string fullPathName)
                 using (var myFileStream = new FileStream(mechanismFullPath, FileMode.Open))
                 {
                     theRobotVariants.mechanism[m] = (mechanism)mySerializer.Deserialize(myFileStream);
+                    myFileStream.Close();
```

Comment:
Close() is not needed. The using statement should take care of this... see https://learn.microsoft.com/en-us/dotnet/api/system.io.filestream?view=net-7.0

Reply:
@muscatd75 The latest commit removed all of these calls.  I was having errors with files being opened by another program and that fixed it in the moment.